man/fi_mr.3.md

```diff
@@ -776,6 +777,27 @@ field is determined by the value specified through iface.
 ## hmem_data
 The hmem_data field is reserved for future use and must be null.
 
+## page_size
+Page size allows applications to notify the provider the page size to be
+used for an MR allocation. Typically, providers can select the optimal
+page size. In cases where a VA range has zero pages backing it, the
+provider may not know the optimal page size during registration. Rather
+than always use a potential inferior page size, allow apps to specify
+the page size to be used.
+
+Page size must be power of 2. If the provider supports the page size
+argument and later determines that this page size cannot be used, RDMA
+operations may fail. For example, suppose the user sets page size to 1
+GiB but the corresponding pages are only 2 MiB. The MR may be configured
+for 1 GiB and the provider cannot change MR page size without
+reconfiguring the MR.
```

Comment:
I'd replace this example with a specific requirement.  "If a specific page_size is specified for a memory region during creation, all pages later associated with the region must be of the given size.  Attaching a memory page of a different size to a region may result in failed transfers to or from the region."